python_examples/NI_example/NI_flow.py

```diff
@@ -91,14 +91,16 @@
 SIRdf["Infected"] = total.sum(axis=1)
 SIRdf = SIRdf.groupby(["time"]).agg(
                                 {"InfectionStatus.Susceptible": 'sum',
+                                 "InfectionStatus.Exposed": 'sum',
                                  "Infected": 'sum',
                                  "InfectionStatus.Recovered": 'sum',
                                  "InfectionStatus.Dead": 'sum'})
 SIRdf.rename(columns={"InfectionStatus.Susceptible": "Susceptible",
+                      "InfectionStatus.Exposed": "Exposed",
                       "InfectionStatus.Recovered": "Recovered"},
              inplace=True)
 
 # Create plot to show SIR curves against time
-SIRdf.plot(y=["Susceptible", "Infected", "Recovered"])
+SIRdf.plot(y=["Susceptible", "Exposed", "Infected", "Recovered"])
```

Comment:
tiny point but should this be renamed as SEIRdf if we now include the exposed compartment?